src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,386 @@
+                /// <exception cref="ArgumentException">This exception can be due to an invalid state.</exception>
+                /// <exception cref="ArgumentNullException">This exception can be due to an invalid null argument.</exception>
+                public QuickPanelClient(TzShell tzShell, Window win, QuickPanelType type)
+                {
+                    if (tzShell == null)
+                    {
+                        throw new ArgumentNullException((string)"tzShell");
+                    }
+                    if (tzShell.GetNativeHandle() == (IntPtr)0)
+                    {
+                        throw new ArgumentException((string)"tzShell is not initialized.");
+                    }
+                    if (win == null)
+                    {
+                        throw new ArgumentNullException((string)"win");
+                    }
+
+                    _tzsh = tzShell;
+                    _tzshWin = win.GetNativeId();
+                    _tzshQpClient = Interop.QuickPanelClient.tzshQuickPanelCreateWithType(_tzsh.GetNativeHandle(), (IntPtr)_tzshWin, (int)type);
+                    if (_tzshQpClient == (IntPtr)0)
+                    {
+                        int err = Tizen.Internals.Errors.ErrorFacts.GetLastResult();
+                        _tzsh.ErrorCodeThrow(err);
+                    }
+                }
+
+                /// <summary>
+                /// Destructor.
+                /// </summary>
+                ~QuickPanelClient()
+                {
+                    Dispose(false);
+                }
+
+                /// <summary>
+                /// Dispose.
+                /// </summary>
+                /// <exception cref="ArgumentException">This exception can be due to an invalid state.</exception>
+                public void Dispose()
+                {
+                    Dispose(true);
+                    GC.SuppressFinalize(this);
+                }
+
+                /// <summary>
+                /// Dispose.
+                /// </summary>
+                /// <exception cref="ArgumentException">This exception can be due to an invalid state.</exception>
+                protected virtual void Dispose(bool disposing)
+                {
+                    int _res;
+                    if (!disposed)
+                    {
+                        if (_tzshQpClient != (IntPtr)0)
+                        {
+                            _res = Interop.QuickPanelClient.tzshQuickPanelDestroy(_tzshQpClient);
+                            _tzsh.ErrorCodeThrow(_res);
+                            _tzshQpClient = (IntPtr)0;
+                        }
+                        disposed = true;
+                    }
+                }
+
+                /// <exception cref="ArgumentException" > This exception can be due to an invalid state.</exception>
```

Comment:
wrong exception comments

Reply:
I changed comment like below
Thrown when failed of invalid argument.
And others exception changed like  "Thrown when~~~"